src/lib/reasoners/fun_sat_frontend.ml

```diff
@@ -39,6 +39,8 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
 
   let empty () = ref (FS.empty ())
 
+  let cdcl_tab_push_pop_stack = Stack.create ()
```

Comment:
This stack cannot be implemented as a global variable; it needs to be part of the environment.

(Also, I don't think it will work since the Tableaux(CDCL) solver uses a mutable CDCL solver internally)